static-analysis/StaticAnalysisChapter.md

```diff
@@ -424,16 +422,18 @@ b = body(e)
     end
     ~~~
 
-2. ~~~.jl
+And then:
+
+~~~.jl
   loops = Int[]
   nesting = 0
   lines = {}
 ~~~
 
-    `loops` is an `Array` of label line numbers where `GoTo`s that are loops occur. `nesting` indicates the number of loops we are currently inside. `lines` is an `Array` of (index, `Expr`) tuples. 
+`loops` is an `Array` of label line numbers where gotos that are loops occur. `nesting` indicates the number of loops we are currently inside. `lines` is an `Array` of (index, `Expr`) tuples. 
```

Comment:
the "gotos" here should be consistent with the "`goto` expressions" above. If "`goto`s" still isn't good, then "`goto` expressions" would be ok.